src/core/default/commands/relationaldb/shell.py

```diff
@@ -23,16 +23,52 @@ def add_arguments(self, parser: ArgumentParser) -> None:
             type=str,
             help="The database to execute on. Name must include component name. ex: comp1.myDb",
         )
+        parser.add_argument(
+            "--c",
+            nargs='+',
+            help="sql command to execute"
+        )
 
     def command(self, *args, **kwargs) -> None:
 
         component_name, database_name = self.get_component_and_resource_from_qualified_name(kwargs.get("resource"))
+        c_command = kwargs.get("c")
+        print(c_command)
         cluster_arn, secret_arn, db_name = get_db_info_from_cdev_name(
             component_name, database_name
         )
-
-        interactive_shell(fmt(Console()), cluster_arn, secret_arn, db_name).cmdloop()
-
+        if c_command != None:
+            print('entrou')
+            c_command = self.concatenate_arg_command(c_command)
+            print(c_command)
+            self.run_sql_command(c_command, cluster_arn, secret_arn, db_name)
+            print('passou')
+        else:
+            interactive_shell(fmt(Console()), cluster_arn, secret_arn, db_name).cmdloop()
+
+    def concatenate_arg_command(self, command):
+        res=''
```

Comment:
string concatenation is extremely inefficient
use ' '.join() 

if item can be int, then use
```
' '.join([str(it) for it in command]
```